docs/getting_started/faq.rst

```diff
@@ -0,0 +1,112 @@
+Frequently Asked Questions (FAQs)
+=================================
+
+Below, you will find answers to the most commonly asked questions about
+Modin. If you still cannot find the answer you are looking for, please post on 
+the #support channel on our Slack_ community or open a Github issue_.
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+While pandas works extremely well on small datasets, as soon as you start working with 
+medium to large datasets that are more than a few GBs, pandas can become painfully 
+slow or run out of memory. This is because pandas is single-threaded. In other words, 
+you can only process your data with one core at a time. This approach does not scale to 
+larger data sets and adding more hardware does not lead to more performance gain. 
+
+The ``modin.pandas`` `DataFrame`_ is a highly scalable, parallel DataFrame. Modin
```

Comment:
the `'DataFrame'_` link here leads to the [DataFrame definition from pandas docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html), is it intended when we're speaking about `modin.pandas DataFrame` here?

Reply:
I took this from the Github README, but I think it should refer to the Modin DataFrame API page, so will change the link.